src/pages/Clients/index.js

```diff
@@ -0,0 +1,251 @@
+                    name="realEstatesConstructionType"
+                    placeholder="Вид на имота"
+                    values={values.realEstatesConstructionType}
+                    options={formatCollectionForReactSelect(realEstateTypes)}
+                    onChange={(element) =>
+                      setFieldValue("realEstatesConstructionType", element)
+                    }
+                  />
+
+                  <FormMultipleSelect
+                    isMulti
+                    className="select-container"
+                    name="realEstatesNeighborhood"
+                    placeholder="Квартали"
+                    values={values.realEstatesNeighborhood}
+                    options={formatCollectionForReactSelect(neighborhoods)}
+                    onChange={(element) =>
+                      setFieldValue("realEstatesNeighborhood", element)
+                    }
+                  />
+                  <FormTextField
+                    variant="filled"
+                    label="Бюджет"
+                    name="budget"
+                    values={values.budget}
+                    handleChange={handleChange}
+                    handleBlur={handleBlur}
+                    errors={errors}
+                  />
+                  <FormTextField
+                    variant="filled"
+                    label="Допълнителна информация"
+                    name="additionalNotes"
+                    values={values.additionalNotes}
+                    handleChange={handleChange}
+                    handleBlur={handleBlur}
+                    errors={errors}
+                  />
+                  {/*footer*/}
+                  <div className="flex items-center justify-end p-6 border-t border-solid border-blueGray-200 rounded-b">
+                    <button
+                      className="text-red-500 background-transparent font-bold uppercase px-6 py-2 text-sm outline-none focus:outline-none mr-1 mb-1 ease-linear transition-all duration-150"
+                      type="button"
+                      onClick={() => setShowForm(false)}
+                    >
+                      Прекрати
+                    </button>
+                    <button
+                      className="bg-green-500 text-white active:bg-emerald-600 font-bold uppercase text-sm px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mr-1 mb-1 ease-linear transition-all duration-150"
+                      type="submit"
+                    >
+                      Добави
+                    </button>
+                  </div>
+                </form>
+              )}
+            </Formik>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function Clients() {
+  const [clientsData, setClientsData] = useState([]);
+  const [clientsLoading, setClientsLoading] = useState(true);
+  const [newClientData, setNewClientData] = useState();
+  const [showForm, setShowForm] = useState(false);
```

Comment:
Please name the boleans to start with 'is' or 'has', so please rename this to something like this.
```suggestion
  const [isFormShowing, setShowForm] = useState(false);
```

Reply:
Changed to the one proposed.